### ISSUE TITLE
can't add a uri record.

### ISSUE THREAD

**Issue by jrwren** [jrwren/domasimu#16]
```
~/bin/domasimu -u 'xmtp.net test1 URI - http://2899904718/ 300'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x642d92]

goroutine 1 [running]:
main.createOrUpdate(0xc000098a00, 0x7ffd51d326cc, 0x2b, 0xc0001cbb53, 0x3, 0x0, 0x444fe6, 0xc0000841cd, 0x6d6f97)
        /home/jrwren/go/src/github.com/jrwren/domasimu/cmd/domasimu/main.go:279 +0x272
main.main()
        /home/jrwren/go/src/github.com/jrwren/domasimu/cmd/domasimu/main.go:97 +0x7c1
```

**Comment by jrwren** [comment]
Because its `URL` not `URI`. Still, it shouldn't panic.

**Comment by jrwren** [comment]
fixed in 2f7175b
